README.md

```diff
@@ -129,8 +126,9 @@ Here is an example of how to kick off a pre-release build:
 
 ```console
 pip install --requirement requirements-dev.txt
-ansible-galaxy install --force --force-with-deps --role-file src/requirements.yml
-AWS_PROFILE=cool-images-ec2amicreate-windows-commando-packer packer build --timestamp-ui -var release_tag=$(./bump_version.sh show) -var is_prerelease=true src/packer.pkr.hcl
+AWS_PROFILE=cool-images-ec2amicreate-windows-server-packer \
+packer build --timestamp-ui -var release_tag=$(./bump_version.sh show) \
+-var is_prerelease=true -var winrm_password="your-winrm-password src/packer.pkr.hcl
```

Comment:
You missed the closing quotes here:
```suggestion
-var is_prerelease=true -var winrm_password="your-winrm-password" src/packer.pkr.hcl
```

Reply:
@dav3r oops.. resolved with [5c8b3fc](https://github.com/cisagov/windows-server-packer/pull/10/commits/5c8b3fcede310bdb320bd9559747919e7f47a770)